### PR TITLE
feat: install extra fonts

### PR DESCRIPTION
**Pull request by aymanbagabas**
Include JetBrains Mono by default with the option to install extra fonts

### CODE REVIEW

File: src/fonts.ts [hunk 1]
```diff
@@ -11,54 +11,97 @@ const specialNames = {
...
+  isExtra: boolean
+}
 
 interface GithubFont {
   owner: string
   repo: string
   assetStartsWith: string
   assetEndsWith: string
   staticPath: string[]
+  isExtra: boolean
```
Comment (maaslalani) [inline src/fonts.ts:25]: For what it's worth, it _might_ be cleaner to name this `isDefault` or `isRequired` so we don't need to add the property to every font. We can only add `isRequired` / `isDefault` to JetBrains Mono and assume everything else is not required if it doesn't have the property defined.

File: README.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ GitHub Action to run [VHS][vhs].
...
-`path`          | Path of the VHS .tape file | ``"vhs.tape"``
-`version`       | Version of VHS to use      | ``"latest"``
-`token`         | GitHub token to use        | ``"${{ github.token }}"``
-`install-fonts` | Whether to install fonts   | ``"true"``
+Name                  | Description                      | Default
+---------------       | --------------------------       | ---------------------
+`path`                | Path of the VHS .tape file       | ``"vhs.tape"``
+`version`             | Version of VHS to use            | ``"latest"``
+`token`               | GitHub token to use              | ``"${{ github.token }}"``
+`install-fonts`       | Whether to install extra fonts   | ``"false"``
```
Comment (maaslalani) [inline README.md:16]:
```suggestion
`install-extra-fonts`       | Whether to install extra fonts   | ``"false"``
```
Did we change this?

File: README.md [hunk 3]
```diff
@@ -80,8 +80,9 @@ jobs:
 
 ## Available Fonts
 
-The action will provide the following fonts (and their [nerd font](nerdfonts)
-variations) by default:
+The action provides `JetBrains Mono` by default. Extra fonts (and their
+[nerd font](nerdfonts) variations) can be installed by setting `install-extra-fonts`
```
Comment (maaslalani) [inline README.md:84]:
```suggestion
[nerd font](nerdfonts) variations) can be installed by setting `install-fonts`
```

There's an inconsistency with the names here. One of them should be changed.

**Review by maaslalani — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (aymanbagabas) [reply README.md:16]: I think keeping `install-fonts` is the right approach to keep backward compatibility.

**Pull request merged**